src/modm/platform/core/stm32/startup_platform.c.in

```diff
@@ -72,3 +73,20 @@ __modm_initialize_platform(void)
 #endif
 %% endif
 }
+
+%% if not core_has_vtor
+void modm_platform_vector_table_remap(void)
+{
+	// Change memory mapped to 0x0 for vector table relocation without VTOR register
+
+	// disable interrupts while remapping memory
+	const uint32_t cpsr = __get_PRIMASK();
+	__disable_irq();
+%% if vector_table_location == "ram"
+	SYSCFG->CFGR1 |= SYSCFG_CFGR1_MEM_MODE;
+%% else
+	SYSCFG->CFGR1 &= ~SYSCFG_CFGR1_MEM_MODE;
+%% endif
+	__set_PRIMASK(cpsr);
+}
+%% endif
```

Comment:
Is there any reason to not put this into `__modm_initialize_platform()`?

Reply:
The idea was to first copy the vector table to ram and then remap the memory but the copying is done in the generic startup part that runs after `__modm_initialize_platform()`. Hm, there shouldn't be any interrupt anyway at that stage. So putting it into the platform specific part could work?